test/services/dumpers/agent-nodejs/expected/mongodb/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.env
```

Comment:
No extra carriage return at the end of our files ?

Reply:
👍